example_app_frames/src/main/java/com/checkout/example/frames/navigation/Screen.kt

```diff
@@ -0,0 +1,7 @@
+package com.checkout.example.frames.navigation
+
+sealed class Screen(val route: String) {
+    object Home : Screen("ckoHomeScreen")
+
+    object DefaultUI : Screen("ckoDefaultUI")
```

Comment:
Can we put these screen identifiers into strings.xml files?

Reply:
I will put them in constants.